.github/workflows/lint.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           go-version: ${{ env.GO_VERSION }}
           check-latest: true
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@774c35bcccffb734694af9e921f12f57d882ef74  # v6.1.1
+        uses: golangci/golangci-lint-action@9937fdf7189f2958a2dc9f6d585e5d65e3326d20  # v6.1.1
```

Comment:
The comment is incorrect?

Reply:
yeah you're right 6.1.1 was released On Oct. 2. Both `774c35...` and `9937fd...` are from main branch.

@apostasie can you explain a bit more the benefit of using sha directly instead of tag (either 6.x.x/6.x/6)? Seems now dependabot will always upgrade to the latest main commit if sha is used.

Reply:
Just looked at commit history. I probably fat fingered the sha copy paste and used a commit from master instead of from the release tag.

Should be / should have been 971e284b6050e8a5849b72094c50ab08da042db8

Reply:
@djdongjin #3777 

Let's try this and see what dependabot is doing.